doc/achievements.md

```diff
@@ -0,0 +1,129 @@
+# Achievement Database Structure
+
+### Id
+Unique achievement ID.
+
+### Group
+Achievement group type.  
+Each achievement type calls a specific objective check.  
+<details class="card">
```

Comment:
Not sure if it is necessary to collapse this list by default or at all.

Reply:
what do you think @aleos89?

Reply:
what do you think @aleos89 ?

Reply:
I'm open to either behavior. Maybe for now we can leave it open (not collapsed) and then in the future if we feel some documents are too long we can implement a collapsing feature across all.

Reply:
Should be good with [a4824a4](https://github.com/rathena/rathena/pull/7975/commits/a4824a4593cd030d997ab412f72bef86e9039f4a)